Roys/app/src/main/java/com/dwg_karrier/roys/Authentication.java

```diff
@@ -65,7 +65,7 @@ static String convertStreamToString(InputStream is) {
   }
 
   void authenticationAndBringPages() {
-    Dialog authDialog = new Dialog(mainContext);
+    final Dialog authDialog = new Dialog(mainContext);
```

Comment:
In onPageFinished function error occurs that authDialog need to be final as that varaiable is accessed from within inner class. 

I'll check it!!